server/meeting_test.go

```diff
@@ -138,7 +138,7 @@ func TestPlugin_GenerateHashtag(t *testing.T) {
 			jsonMeeting, err := json.Marshal(tt.args.meeting)
 			tAssert.Nil(err)
 			api.On("KVGet", tt.args.meeting.ChannelID).Return(jsonMeeting, nil)
-			got, err := mPlugin.GenerateHashtag(tt.args.meeting.ChannelID, tt.args.nextWeek, -1)
+			got, err := mPlugin.GenerateHashtag(tt.args.meeting.ChannelID, tt.args.nextWeek, -1, false, time.Now().Weekday()) // last parameter doesn't matter unless requeue
```

Comment:
Can we change the last parameter to not rely on `time.Now()`. Tests should have completely static data if possible. The comment here can be removed too.

Reply:
Not sure about the best way to do same, since go lang doesn't support default values for parameters.
any ideas?

Reply:
Just use some random hardcoded day as the test data, and assert expected output based on that static input

Reply:
yes, did same in new PR.